src/radical/pilot/agent/lm/jsrun.py

```diff
@@ -87,7 +87,7 @@ def _create_resource_set_file(self, slots, uid, sandbox):
                 rs_str += ' cpu: {%s}'  % cores
                 if gpu_maps:
                     gpus  = ','.join(str(gpu) for gpu in gpu_maps.pop(0))
-                    rs_str += '; gpu: {%s}' % gpus
+                    rs_str += '; gpu: {%s}' % gpus if gpus != '' else ''
                 rs_str += '}\n'
```

Comment:
Did you find why the `gpu_maps` contained an empty list?

Reply:
@lee212 : ping?

Reply:
I wonder what expected `gpu_maps` looks like. For example, `[[]]` was presented but `if gpu_maps:` condition couldn't detect this as an empty which I believe this is a valid case, isn't it?

Reply:
I do not find why `gpu_map` is presented with an empty list.